appinventor/misc/mavendl/mavendl.py

```diff
@@ -0,0 +1,128 @@
+# Create mavendl to download AAR libs with transitive deps 
+# This stores the files in a "libs" directory relative to this script
+# Usage: mavendl.py <group>:<artifact>:<version>
+
+import argparse
+import os
+import urllib.request
+from io import BytesIO
+from typing import Dict, List, Set, Tuple
+from xml.dom import minidom
+from zipfile import ZipFile
+
+parser = argparse.ArgumentParser()
+parser.add_argument("coord")
+
+args = parser.parse_args()
+
+
+maven_repos = [
+    # google
+    'https://maven.google.com',
+    # maven central
+    'https://repo1.maven.org/maven2',
+]
+
+
+def maven_url(repo: str, group: str, artifact: str, version: str, type: str) -> str:
+    return '{r}/{gp}/{a}/{v}/{a}-{v}.{t}'.format(r=repo, gp=group.replace('.', '/'), a=artifact, v=version, t=type)
```

Comment:
If targeting Python 3.6 or later, consider switching to f-strings.

Reply:
I just used format for readability